vic/drivers/shared_all/src/initialize_veg.c

```diff
@@ -39,31 +39,33 @@ initialize_veg(veg_var_struct **veg_var,
 
     for (i = 0; i < Nveg; i++) {
         for (j = 0; j < options.SNOW_BAND; j++) {
-            veg_var[i][j].Wdew = 0.0;
-            veg_var[i][j].throughfall = 0.0;
+            veg_var[i][j].albedo = 0.0;
+            veg_var[i][j].displacement = 0.0;
+            veg_var[i][j].fcanopy = 0.0;
             veg_var[i][j].LAI = 0.0;
+            veg_var[i][j].roughness = 0.0;
+            veg_var[i][j].throughfall = 0.0;
+            veg_var[i][j].Wdew = 0.0;
             veg_var[i][j].Wdmax = 0.0;
-            veg_var[i][j].fcanopy = 0.0;
-            veg_var[i][j].albedo = 0.0;
```

Comment:
It seems like it should.  However, I would suggest that we create a separate issue for that - an issue that would list all the things we need to change to make restarts be exact.  This PR is only for the purpose of allowing us to remove instances of `month` from vic_run().